Dockerfile

```diff
@@ -1,10 +1,10 @@
 FROM alpine:3.7
-ARG VERSION=1.9.0
+ARG VERSION=1.11.0
 
 # Inspiration from https://github.com/gmr/alpine-pgbouncer/blob/master/Dockerfile
 RUN \
   # Download
-  apk --update add autoconf autoconf-doc automake udns udns-dev curl gcc libc-dev libevent libevent-dev libtool make man libressl-dev pkgconfig postgresql-client && \
```

Comment:
Do we actually need to install bash inside of container by default? I'm afraid that this will only increase the size of the container, but will bring very little benefit.

Reply:
No we dont need but we often use bash as interpreter and that's why i leave it. If you don't like I can remove it!

Reply:
Of course I like bash. I have no right to forbid you to add bash :) I think, probably, at least it’s better to do it in a separate pull request, and keep this pull request focused on version bump.

Reply:
Already removed it